src/plugins/data/common/osd_field_types/types.ts

```diff
@@ -62,7 +62,7 @@ export enum OPENSEARCH_FIELD_TYPES {
   HALF_FLOAT = 'half_float',
   SCALED_FLOAT = 'scaled_float',
   DOUBLE = 'double',
-  INTEGER = 'integer',
+  INTEGER = 'int', // opensearch has changed type from 'integer' to 'int'
```

Comment:
@abbyhu2000 I approved this PR, feel free to update it in your next PR:
```suggestion
  INTEGER = 'integer',
  INT = 'int',
```